multiworld/world_communicator.py

```diff
@@ -142,7 +164,18 @@ async def recv(
     async def broadcast(
         self, tensor: Tensor, src: int, world_name: str = DEFAULT_WORLD_NAME
     ) -> None:
-        """Broadcast a tensor to the world from a source (src)."""
+        """
+        Broadcast a tensor from a source (src) to all other ranks in the same world.
+
+        Args:
+            tensor: Tensor to be broadcast.
+            src: Source of the broadcast.
+            world_name: Name of the world.
+
+        Raises:
+            BrokenWorldException: An error that occurs when
+            the world is broken due to worker, node or network failure.
```

Comment:
```suggestion
                the world is broken due to worker, node or network failure.
```
Please add 4 spaces for the rest of methods too.